src/macros.rs

```diff
@@ -2,7 +2,16 @@ macro_rules! json {
     ($input:ident) => {{
         match serde_json::to_vec(&$input) {
             Ok(data) => data,
-            Err(err) => return Box::new(future::err(err.into())),
+            Err(err) => return Box::new(futures::future::err(err.into())),
         }
     }};
 }
+
+macro_rules! json_lit {
```

Comment:
I kind of regret using `json!` since I later discovered it was in serde_json. I'm cool with this though

Reply:
Well, this removes the boilerplate of early returning too, which it's a shame can't be abstracted without a macro.